projects/storefrontlib/src/cms-structure/page/config/default-cart-page.config.ts

```diff
@@ -1,4 +1,21 @@
-import { CmsPageConfig } from '@spartacus/core';
+import { CmsPageConfig, ContentSlotComponentData } from '@spartacus/core';
+
+export const cartComponents: {
+  [key: string]: ContentSlotComponentData | any;
```

Comment:
What about extracting this to separate type for it, because we use in many places?

Reply:
I leave that for later.